frontend/src/components/Board/Board.js

```diff
@@ -21,26 +42,57 @@ export default class Board extends React.Component {
     }));
   };
 
+  updateBoard = (row, column, value) => {
+    this.setState(prev => ({
+      boardModel: _.set(
+        prev.boardModel,
+        `rows['${row}'].['${column}'].value`,
+        value
+      )
+    }));
+    this.hideSuggestions();
+  };
+
   render() {
+    // const pressedFieldStyle = {backround: this.state.suggestions ? "green" : "purple"};
```

Comment:
Don't merge commented code unless there is a reason for it. And if there is a reason add a comment explaining what that reason is.